DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: teal.code
 Title: Code Storage and Execution Class for 'teal' Applications
-Version: 0.4.1.9024
+Version: 0.5.0
```

Comment:
`teal.code` went on CRAN with version `0.4.2`. Shouldn't we release on GitHub with `0.4.2` also?